doc/development/development_on_osx.rst

```diff
@@ -1,36 +1,24 @@
-This section contains information about setting up a Tribler development environment on macOS. Unlike Linux based systems where installing third-party libraries is often a single ``apt-get`` command, installing and configuring the necessary libraries requires more attention on macOS. This guide has been tested with macOS 10.10.5 (Yosemite) but should also work for macOS 10.11 (El Capitan).
+Tribler development environment setup on MacOS (10.10 or 10.11).
 
-Note that the guide below assumes that Python is installed in the default location of Python (shipped with macOS). This location is normally in ``/Library/Python/2.7``. Writing to this location requires root access when using easy_install or pip. To avoid root commands, you can install Python in a virtualenv. More information about setting up Python in a virtualenv can be found `here <http://www.marinamele.com/2014/05/install-python-virtualenv-virtualenvwrapper-mavericks.html>`_.
+1. MacPorts
+2. HomeBrew 
+3. Build
+4. Notes
 
-Introduction
-------------
+`MacPorts <https://www.macports.org>`_
+======================================
 
-Compilation of C/C++ libraries should be performed using Clang which is part of the Xcode Command Line Tools. The Python version shipped with macOS can be used and this guide has been tested using Python 2.7. The current installed version and binary of Python can be found by executing:
+.. code-block:: bash
 
-.. code-block:: none
-
-    python --version # gets the python version
-    which python # prints the path of the Python executable
-
-Note that the default location of third-party Python libraries (for example, installed with ``pip``) can be found in ``/Library/Python/2.7/site-packages``.
-
-Many packages can be installed by using the popular brew and pip executables. Brew and pip can be installed by using:
-
-.. code-block:: none
-
-    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
Code duplication is bad, I added a link to the official instructions (in case they change etc)